examples/ts/btc/lightning/custodial-wallet.ts

```diff
@@ -0,0 +1,100 @@
+/**
+ * Create a Lightning custodial wallet at BitGo.
+ * 
+ * IMPORTANT: Your BitGo account must have the "custodyLightningWallet" license
+ * enabled to use this functionality. Contact BitGo support if you receive a
+ * license-related error.
+ * 
+ * Copyright 2025, BitGo, Inc.  All Rights Reserved.
+ */
+
+import { BitGoAPI } from '@bitgo/sdk-api';
+import { Tlnbtc } from '@bitgo/sdk-coin-lnbtc';
+require('dotenv').config({ path: '../../.env' });
+
+// TODO: set access token for testnet
+// Get this from your BitGo account
+const accessToken = process.env.TESTNET_ACCESS_TOKEN || '';
+
+// TODO: set passphrase to create a wallet
+const passphrase = process.env.WALLET_PASSPHRASE || '';
+
+// TODO: set your enterprise ID
+const enterprise = process.env.ENTERPRISE_ID || '';
+
+// Generate a passcode encryption code (required for Lightning wallets)
+// IMPORTANT: Store this information securely. You will need it to recover your wallet if you lose wallet password.
+const passcodeEncryptionCode = process.env.PASSCODE_ENCRYPTION_CODE || '';
+
+// Use tlnbtc for testnet, lnbtc for mainnet
+const coin = 'tlnbtc';
+
+/**
+ * Create a Lightning custodial wallet
+ * This function creates a custodial Lightning wallet on the BitGo platform
+ * @returns {Promise<any>} Wallet object
+ */
+async function main(): Promise<any> {
```

Comment:
```suggestion
async function main(): Promise<void> {
```
without the need of return at the end